factory/.env

```diff
@@ -11,7 +11,7 @@ BASE_IMAGE='debian:bullseye-slim'
 NODE_VERSION='16.18.1'
 
 # Update this to deploy the docker factory if you make changes to factory.Dockerfile or install scripts
-FACTORY_VERSION='1.0.1'
```

Comment:
Should we add a changelog for the factory image?

Reply:
I think thats a good idea, let me whip something up.